### PR TITLE
edgecore: add nfs localpath support

### PR DESCRIPTION
**Pull request by swartz-k**
Signed-off-by: swartz-k <9215868@gmail.com>

**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:
Support use PV of localpath and nfs in edgecore

User can use like this
```
apiVersion: apps/v1
kind: StatefulSet
metadata:
......
  volumeClaimTemplates:
  - metadata:
      name: data
    spec:
      accessModes: [ "ReadWriteOnce" ]
      storageClassName: "local-path"
      resources:
        requests:
          storage: 10Gi
```

### CODE REVIEW

File: edge/pkg/edged/edged.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"k8s.io/kubernetes/pkg/volume/local"
+	"k8s.io/kubernetes/pkg/volume/nfs"
```
Comment (fisherxu) [inline edge/pkg/edged/edged.go:31]: Please sort the imports :)

Comment (swartz-k) [reply edge/pkg/edged/edged.go]: Resolved and tested `make verify-vendor`

**Comment by fisherxu** [comment]
Looks like the failed ut is not related to this pr, I will have a look

File: .gitignore [hunk 2]
```diff
@@ -16,3 +16,4 @@ edgesite/edgesite
 keadm/keadm
 _output
 _tmp
+build/edge/tmp
```
Comment (daixiang0) [inline .gitignore:19]: which generates it?

**Comment by fisherxu** [comment]
@swartz-k Please rebase the master branch and let's see the ut result.

File: .gitignore (hunk 2 above)
Comment (swartz-k) [reply .gitignore:19]: [Makefile#L278](https://github.com/kubeedge/kubeedge/blob/2f4447c60b767de371a8b773774662711ead3089/Makefile#L278)`edgeimage`

**Comment by swartz-k** [comment]
> @swartz-k Please rebase the master branch and let's see the ut result.

Thanks a lot. It passed.

**Comment by fisherxu** [comment]
/lgtm
/approve
Thanks for the enhancement！

**Pull request merged**